### PR TITLE
Misc documentation and comment updates

### PR DESCRIPTION
**Pull request by alistair-singh**
Snowbridge: https://github.com/Snowfork/snowbridge/pull/1076

### CODE REVIEW

File: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs [hunk 1]
```diff
@@ -388,6 +388,8 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
 
+/// Creates an AgentId from a MultiLocation. An AgentId is a unique mapping to a Agent contract
+/// which acts as the sovereign account for the MultiLocation.
```
Comment (claravanstaden) [inline cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs:392]:
```suggestion
/// Creates an AgentId from a MultiLocation. An AgentId is a unique mapping to a Agent contract on Ethereum
/// which acts as the sovereign account for the MultiLocation.
```

**Review by claravanstaden — APPROVED** [review]

**Comment by claravanstaden** [comment]
@alistair-singh maybe if you have time you could do this one too? https://github.com/paritytech/polkadot-sdk/pull/2522#discussion_r1431385493

File: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs (hunk 1 above)
Comment (alistair-singh) [reply cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs]: Resolved in https://github.com/Snowfork/polkadot-sdk/pull/81/commits/0d33404e71db15120519566dfade77dd36bd477c

**Comment by alistair-singh** [comment]
> @alistair-singh maybe if you have time you could do this one too? [paritytech#2522 (comment)](https://github.com/paritytech/polkadot-sdk/pull/2522#discussion_r1431385493)

Addressed as many as I could find included in this PR.

**Pull request merged**